### DISCUSSION TITLE
我已经付了340元2024年费用，蓝灯突然变成免费版

### DISCUSSION THREAD

**Discussion by jinrumyLw611112** [getlantern/lantern-forum-cn#17] [General]
你们的邮件我已经收到，根据你们的建议重新下载安装最新的蓝灯，常见问题上解答页面找不到介绍的部分，但问题照旧，还是免费版，看来只有你们内部技术部门查找原因解决。
我从2020开始使用你们的产品，2022年11月的时候我电脑坏了换了电脑，结果也是专业版变成了免费版，再付340元后才变成专业版，也就是2023年我的专业版蓝灯实际付了340*2=680元。
2023/12/12在到期前一天我就续费340元后，一直是专业版使用正常，到2023/12/28下午莫名其妙的变成了免费版的了，这严重影响我的使用，也损失了你们的信誉，不可思议。
做生意诚信最重要，请你们内部查找原因吧，我这种情况肯定不是唯一的，我把付费记录都提供给你们了，你们查不出来吗？你们实在不能解决请把我付的340元原路退回吧。每次收到冷冰冰的机器人回复，什么问题都没有解决，解决很困难吗？小客户你们就漠视吗？小客户没有付钱吗？

**Comment by MaxMKMax** [comment]
对于您面临的问题，我们深表歉意。我是新加入的客服。从现在开始，我们有中文客服来支持您的问题。我们目前有免费版和专业版。我可以了解您所面临问题的更多详细信息吗？所以我可以直接帮助您解决问题。与免费版本相比，您购买的付费版本可能具有更多功能。

如果您在使用专业版功能时遇到问题，请告诉我。我会帮你解决这个问题。谢谢您!

**Reply by MaxMKMax** [reply]
您好! 我已经将您的问题联系了我们的团队。您可以发送电子邮件至 support@getlantern.org 吗？其中需要包含购买证明和您购买 Lantern 时用的电子邮件。然后我们会进行处理。谢谢!

**Reply by Phranoid** [reply]
蓝灯经常连不上怎么解决啊

**Reply by MaxMKMax** [reply]
@Phranoid 您有邮箱还是电子钱包的支付记录吗？或者购买单也可以。然后直接用您的QQ号或邮箱系统直接发邮件给support@getlantern.org就行了。

您可以在邮箱里说明情况和写上您的登陆邮箱。谢谢。再次和您抱歉让您遇上麻烦了。

**Comment by mathxiaobin** [comment]
我也遇到了类似的问题，我23年大概10月前后付过两次会员的钱，但是都没有在会员时间上有所体现，以为只是网页显示问题，但是这几天直接变成了免费版

**Reply by MaxMKMax** [reply]
好的，理解了。我们会重视这个问题并尽快解决问题。您有邮箱还是电子钱包的支付记录吗？或者购买单也可以。然后直接用您的QQ号或邮箱系统直接发邮件给 [support@getlantern.org] 就行了。

您可以在邮箱里说明情况和写上您的登陆邮箱。谢谢。再次和您抱歉让您遇上麻烦了。

**Comment by flcode** [comment]
我被扣费两次，但是只有一年的时间！请把我被多扣的钱返还！

**Reply by MaxMKMax** [reply]
@flcode 您可以向我提供您购买时的电子邮件地址吗？我帮您要求付款团队核实，核实后我们会帮您安排另一年的退款。谢谢。

**Comment by flcode** [comment]
好的，我的电子邮件 ***@***.***发自我的 iPhone在 2024年2月9日，00:42，Mike K ***@***.***> 写道：﻿
@flcode 您可以向我提供您购买时的电子邮件地址吗？我帮您要求付款团队核实，核实后我们会帮您安排另一年的退款。谢谢。

**Reply by MaxMKMax** [reply]
@flcode 你好。抱歉，我无法看到您的电子邮件地址，因为您的电子邮件已被隐藏。它显示****@*** 您可以重新发送一次吗？谢谢!

**Comment by flcode** [comment]
***@***.***发自我的 iPhone在 2024年2月9日，18:24，Mike K ***@***.***> 写道：﻿
@flcode 你好。抱歉，我无法看到您的电子邮件地址，因为您的电子邮件已被隐藏。它显示****@*** 您可以重新发送一次吗？谢谢!

**Comment by jayron-git** [comment]
我也是一樣的狀況，1月底就出現這樣的狀況，但是寫信好幾次給support@lantern.io  
都只能得到罐頭訊息，都是沒有用的狀況
我也說明我的付費狀況，可是到現在都還沒有人回覆
我2024年12月27日以mastercard付款的，金額是87.0 USD
我現在該怎麼辦，已經超過半個月都是顯示藍燈費版

> 对于您面临的问题，我们深表歉意。我是新加入的客服。从现在开始，我们有中文客服来支持您的问题。我们目前有免费版和专业版。我可以了解您所面临问题的更多详细信息吗？所以我可以直接帮助您解决问题。与免费版本相比，您购买的付费版本可能具有更多功能。
> 
> 如果您在使用专业版功能时遇到问题，请告诉我。我会帮你解决这个问题。谢谢您!

**Comment by MaxMKMax** [comment]
@jayron-git 您好。我们的客户支持电子邮件是support@getlantern.org , 不是 support@lantern.io
请您给我您的邮箱. 我帮您让团队给您手动升级去专业版. 不好意思让您等了. 谢谢您!

**Reply by jayron-git** [reply]
您好  上週課服有回信給我，說我的email已經是pro account，但是我打開lantern仍然是free account! 
我也回信截圖給課服，至今仍未有回應，我打開lantern的畫面如下
![2](https://github.com/getlantern/lantern-forum-cn/assets/62708886/4a8c26ae-f174-4b8c-b5a7-b4fd1173c1d2)

**Comment by jayron-git** [comment]
您好，已經寫email到支持電子郵件[support@getlantern.org](mailto:support@getlantern.org)
並且附上刷卡證明的截圖，若有收到再麻煩您回覆

> @jayron-git 您好。我们的客户支持电子邮件是[support@getlantern.org](mailto:support@getlantern.org) , 不是 [support@lantern.io](mailto:support@lantern.io) 请您给我您的邮箱. 我帮您让团队给您手动升级去专业版. 不好意思让您等了. 谢谢您!

**Comment by MaxMKMax** [comment]
@jinrumyLw611112 好的谢谢您. 我再帮您跟进一下.

**Reply by xiangqili** [reply]
你好 我在2024-01-23 12:16:07 已经付了340元2024年费用，今天蓝灯突然变成免费版,客服邮箱也发不出去。

**Comment by xiangqili** [comment]
你好 我在2024-01-23 12:16:07 已经付了340元2024年费用，今天蓝灯突然变成免费版,客服邮箱也发不出去。

**Reply by MaxMKMax** [reply]
@xiangqili 您好, 请您提供我您的账号用的的邮件。我帮您提交您的问题给支付团队。他们会手动帮助您升级到专业版。非常抱歉影响了您的体验感, 谢谢您对蓝灯的支持。

**Comment by xiangqili** [comment]
你好&nbsp;
我的账号邮件是 ***@***.***


------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "getlantern/lantern-forum-cn"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2024年2月21日(星期三) 晚上10:01
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [getlantern/lantern-forum-cn] 我已经付了340元2024年费用，蓝灯突然变成免费版 (Discussion #17)





 
@xiangqili 您好, 请您提供我您的账号用的的邮件。我帮您提交您的问题给支付团队。他们会手动帮助您升级到专业版。非常抱歉影响了您的体验感, 谢谢您对蓝灯的支持。

**Comment by xiangqili** [comment]
你好
我的账号邮箱是 ***@***.***
谢谢




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "getlantern/lantern-forum-cn"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2024年2月21日(星期三) 晚上10:01
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [getlantern/lantern-forum-cn] 我已经付了340元2024年费用，蓝灯突然变成免费版 (Discussion #17)





 
@xiangqili 您好, 请您提供我您的账号用的的邮件。我帮您提交您的问题给支付团队。他们会手动帮助您升级到专业版。非常抱歉影响了您的体验感, 谢谢您对蓝灯的支持。

**Reply by MaxMKMax** [reply]
@xiangqili 您好. 好的. 我帮您提交上去. 您的邮箱被屏蔽了. 您把邮箱分开着写好吗? 谢谢您!
因为我看到这样的东西 ***@***.***

**Reply by xiangqili** [reply]
邮箱发不出去

**Reply by xiangqili** [reply]
你发给我邮箱收件人的地址就是 我的账号邮箱

**Comment by LaputaZeej** [comment]
去年7月份购买的，现在登陆邮箱一直收不到验证码 。账号：leafcloudland @ 163.com

**Reply by MaxMKMax** [reply]
@LaputaZeej 
您好，谢谢您的反馈。蓝灯已经在正视这个问题了。 我们正在修复问题和进行优化。请您耐心等待。非常抱歉影响了您的体验感。谢谢您的等待与支持。

您可以帮助我们在应用程序中提交报告您的问题吗？您可以访问Lantern应用程序并提交log/ 报告。

1. go to Support (转到支持)
2. select Report an Issue (选择报告问题)
3. choose issue type (选择问题类型)
4. include additional details (包括额外的详细信息)
5. hit Send Report (点击发送报告)

当您在app提交问题报告后您会收到邮件. 邮件标题上有 8 位数号码. 请回来这里留言您的log/ticket号码. 因为我需要拿您的反馈报告号码来和科技团队沟通您遇到的问题. 不然我无法提交您遇到的问题给科技团队.

有了这些资料，我们科技团队便可以尽快解决问题。您可以在留言区给我知道。谢谢。
我们将非常感谢您。

**Comment by xiangqili** [comment]
你好
***@***.***
谢谢


------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "getlantern/lantern-forum-cn"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2024年2月22日(星期四) 下午2:06
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [getlantern/lantern-forum-cn] 我已经付了340元2024年费用，蓝灯突然变成免费版 (Discussion #17)





 
@xiangqili 您好. 好的. 我帮您提交上去. 您的邮箱被屏蔽了. 您把邮箱分开着写好吗? 谢谢您!
 因为我看到这样的东西 @.***

**Comment by LaputaZeej** [comment]
address :

l&nbsp; e&nbsp; a f c&nbsp; l o u d l a n d

@

1 6&nbsp; &nbsp; 3&nbsp; &nbsp;.&nbsp; &nbsp;c&nbsp; &nbsp; o&nbsp; m




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "getlantern/lantern-forum-cn"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2024年2月27日(星期二) 下午3:54
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [getlantern/lantern-forum-cn] 我已经付了340元2024年费用，蓝灯突然变成免费版 (Discussion #17)





 你好 
 ***@***.*** 
 谢谢 
 
 
 ------------------&amp;nbsp;原始邮件&amp;nbsp;------------------ 
 发件人:                                                                                                                        "getlantern/lantern-forum-cn"                                                                                    ***@***.***&amp;gt;; 
 发送时间:&amp;nbsp;2024年2月22日(星期四) 下午2:06 
 ***@***.***&amp;gt;; 
 ***@***.******@***.***&amp;gt;; 
 主题:&amp;nbsp;Re: [getlantern/lantern-forum-cn] 我已经付了340元2024年费用，蓝灯突然变成免费版 (Discussion #17) 
 
 
 
 
 
   
 @xiangqili 您好. 好的. 我帮您提交上去. 您的邮箱被屏蔽了. 您把邮箱分开着写好吗? 谢谢您! 
  因为我看到这样的东西 @.***

**Comment by LaputaZeej** [comment]
number is :&nbsp;69736934



------------------&nbsp;原始邮件&nbsp;------------------
发件人: "Mike ***@***.***&gt;; 
发送时间: 2024年2月28日(星期三) 凌晨0:02
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [getlantern/lantern-forum-cn] 我已经付了340元2024年费用，蓝灯突然变成免费版 (Discussion #17)





 
@LaputaZeej
 您好，谢谢您的反馈。蓝灯已经在正视这个问题了。 我们正在修复问题和进行优化。请您耐心等待。非常抱歉影响了您的体验感。谢谢您的等待与支持。
 
您可以帮助我们在应用程序中提交报告您的问题吗？您可以访问Lantern应用程序并提交log/ 报告。
  
go to Support (转到支持)
 
select Report an Issue (选择报告问题)
 
choose issue type (选择问题类型)
 
include additional details (包括额外的详细信息)
 
hit Send Report (点击发送报告)
  
当您在app提交问题报告后您会收到邮件. 邮件标题上有 8 位数号码. 请回来这里留言您的log/ticket号码. 因为我需要拿您的反馈报告号码来和科技团队沟通您遇到的问题. 不然我无法提交您遇到的问题给科技团队.
 
有了这些资料，我们科技团队便可以尽快解决问题。您可以在留言区给我知道。谢谢。
 我们将非常感谢您。

**Comment by BinBinMind** [comment]
@Mike777Khoo 你好，我的专业版马上到期，续费使用国内银联卡都显示无法支付。有没有支付宝之类的其他支付渠道（没有VISA或者百事通）

**Reply by MaxMKMax** [reply]
@BinBinMind 您好。您可以用这个链接支付。这个链接可以用支付宝! 谢谢您的支持。非常开心有您这么尊贵的用户。

https://buy.stripe.com/eVacPZe3A5cp1oc4gO

**Reply by tudour3** [reply]
@Mike777Khoo  可以发一个1年的链接吗？

**Reply by MaxMKMax** [reply]
@tudour3 
您好. 我们暂时只有这个链接. 非常抱歉. 要不我再帮您问一下.

**Reply by MaxMKMax** [reply]
@tudour3 
https://buy.stripe.com/7sI8zJ4t00W97MA9Bz
您好. 我帮您要到一年专业版配套的链接了. 希望这可以帮到您. 谢谢您的支持!

**Reply by tudour3** [reply]
@Mike777Khoo  好的 感谢

**Reply by bravoroo** [reply]
工单号#160990，麻烦帮我看一下，等了差不多一天了没回复
